src/release/main.d

```diff
@@ -21,8 +21,10 @@ import release.mergeHelper;
 import semver.Version;
 
 import octod.api.repos;
+import octod.api.releases;
```

Comment:
I guess this change sneaked in this commit accidentally?

Reply:
No. The new way makd compiles my main.d file now makes dmd miss that import and that leads to linker errors. Importing it not as nested but instead on file scope fixes that